docs/python/tutorial-azure-functions.md

```diff
@@ -0,0 +1,348 @@
+
+    - Template: HTTP trigger
+    - Name: "DigitsOfPi"
+    - Authorization level: Anonymous
+
+1. In the VS Code file explorer, you should see a subfolder for your function name that again contains files named `__init__.py`, `function.json`, and `sample.dat`.
+
+1. Replace the code in `__init__.py` to match the following, which generates a string containing the value of PI to a number of digits specified in the URL (this code uses only a URL parameter)
+
+    ```python
+    import logging
+
+    import azure.functions as func
+
+    """ Adapted from the second, shorter solution at http://www.codecodex.com/wiki/Calculate_digits_of_pi#Python
+    """
+
+    def pi_digits_Python(digits):
+        scale = 10000
+        maxarr = int((digits / 4) * 14)
+        arrinit = 2000
+        carry = 0
+        arr = [arrinit] * (maxarr + 1)
+        output = ""
+
+        for i in range(maxarr, 1, -14):
+            total = 0
+            for j in range(i, 0, -1):
+                total = (total * j) + (scale * arr[j])
+                arr[j] = total % ((j * 2) - 1)
+                total = total / ((j * 2) - 1)
+
+            output += "%04d" % (carry + (total / scale))
+            carry = total % scale
+
+        return output;
+
+    def main(req: func.HttpRequest) -> func.HttpResponse:
+        logging.info('DigitsOfPi HTTP trigger function processed a request.')
+
+        digits_param = req.params.get('digits')
+
+        if digits_param is not None:
+            try:
+                digits = int(digits_param)
+            except ValueError:
+                digits = 10   # A default
+
+            if digits > 0:
+                digit_string = pi_digits_Python(digits)
+
+                # Insert a decimal point in the return value
+                return func.HttpResponse(digit_string[:1] + '.' + digit_string[1:])
+
+        return func.HttpResponse(
+             "Please pass the URL parameter ?digits= to specify a positive number of digits.",
+             status_code=400
+        )
+    ```
+
+1. Because the code supports only HTTP GET, modify `functions.json` so that the `"methods"` collection contains only `"get"` (that is, remove `"post"`).
+
+1. Start the debugger by pressing F5 or selecting the **Debug** > **Start Debugging** menu command. The output window should now show both endpoints in your project:
+
+    ```
+    Http Functions:
+
+            DigitsOfPi: [GET] http://localhost:7071/api/DigitsOfPi
+
+            HttpExample: [GET,POST] http://localhost:7071/api/HttpExample
+    ```
+
+1. In a browser, or from curl, make a request to `http://localhost:7071/api/DigitsOfPi?digits=125` and observe the output. (You might notice that the code algorithm isn't entirely accurate, but we'll leave the improvements to you!) Stop the debugger when you're finished.
```

Comment:
COPY (clarity, style): Remove parenthetical

Reply:
With casual VS Code tone, I think it's appropriate because otherwise someone's going to pick on that particular detail. If we don't note if ourselves, people are left to ask whether we noticed at all. This way is better, then.